core/src/main/java/com/graphhopper/routing/util/parsers/OSMRoadClassParser.java

```diff
@@ -43,8 +42,8 @@ public void createEncodedValues(EncodedValueLookup lookup, List<EncodedValue> li
     }
 
     @Override
-    public IntsRef handleWayTags(IntsRef edgeFlags, ReaderWay readerWay, EncodingManager.Access access, IntsRef relationFlags) {
-        if (!access.isWay())
+    public IntsRef handleWayTags(IntsRef edgeFlags, ReaderWay readerWay, boolean ferry, IntsRef relationFlags) {
+        if (ferry)
```

Comment:
This is the actual fix.

Reply:
Ah I see, that explains the issue :+1:.

Reply:
BTW: do we need to pass the ferry boolean? We pass the that should contain the ferry information? Or is this for some ferry relations or something like this?